src/main/java/org/broadinstitute/hellbender/transformers/BaseQualityClipReadTransformer.java

```diff
@@ -1,17 +1,18 @@
 package org.broadinstitute.hellbender.transformers;
 
+import org.broadinstitute.hellbender.utils.clipping.ClippingOp;
+import org.broadinstitute.hellbender.utils.clipping.ClippingRepresentation;
+import org.broadinstitute.hellbender.utils.clipping.ReadClipper;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
-import java.util.Arrays;
-
 /**
  * Clips reads on both ends using base quality scores
  */
 public class BaseQualityClipReadTransformer implements ReadTransformer {
```

Comment:
I forgot to mention that you probably intended this class to be `final` too, and its `serialVersionUID` to be `private`.